redash/settings/__init__.py

```diff
@@ -141,6 +141,11 @@ def all_settings():
 MAIL_MAX_EMAILS = os.environ.get('REDASH_MAIL_MAX_EMAILS', None)
 MAIL_ASCII_ATTACHMENTS = parse_boolean(os.environ.get('REDASH_MAIL_ASCII_ATTACHMENTS', 'false'))
 
+
+def email_server_is_configured():
```

Comment:
Figured it's best to hide this instead of relying on `MAIL_DEFAULT_SENDER` in different places in the app, in case we ever want to change the criteria.